test/integration/fuzz/lockup/withdraw.t.sol

```diff
@@ -206,13 +155,13 @@ abstract contract Withdraw_Integration_Fuzz_Test is Integration_Test, Withdraw_I
     /// - Multiple withdraw amounts
     function testFuzz_Withdraw(
         uint256 timeJump,
+        address caller,
```

Comment:
Why'd you add caller here? There is no prank to caller address in this test function.

Reply:
added by mistake